install/windows/build.py

```diff
@@ -61,34 +62,26 @@ def build_odemisviewer_inst():
     nsis_cmd = nsis_command[:-1] + info + [nsis_command[-1]]
     return run_command(nsis_cmd, "odemis")
 
-
-def build_delphiviewer_inst():
-    info = [
-        "/DPRODUCT_NAME=DelphiViewer",
-        "/DPRODUCT_HNAME=Delphi Viewer",
-        "/DIMAGE=install_delphi.bmp",
-        "/DWEBSITE=http://www.delphimicroscope.com",
-    ]
-    nsis_cmd = nsis_command[:-1] + info + [nsis_command[-1]]
-    run_command(nsis_cmd, "delphi")
+# def build_delphiviewer_inst():
+#     info = [
+#         "/DPRODUCT_NAME=DelphiViewer",
+#         "/DPRODUCT_HNAME=Delphi Viewer",
+#         "/DIMAGE=install_delphi.bmp",
+#         "/DWEBSITE=http://www.delphimicroscope.com",
+#     ]
+#     nsis_cmd = nsis_command[:-1] + info + [nsis_command[-1]]
+#     run_command(nsis_cmd, "delphi")
```

Comment:
Either undelete this function or add a comment why it should remain commented and not deleted